catalyst/rl/offpolicy/algorithms/core.py

```diff
@@ -220,23 +224,23 @@ def prepare_for_trainer(cls, config):
         n_step = config_["shared"]["n_step"]
         gamma = config_["shared"]["gamma"]
         history_len = config_["shared"]["history_len"]
-        trainer_state_shape = (config_["shared"]["observation_size"], )
-        trainer_action_shape = (config_["shared"]["action_size"], )
+        trainer_state_shape = (config_["shared"]["observation_size"],)
+        trainer_action_shape = (config_["shared"]["action_size"],)
 
-        actor_fn = config_["actor"].pop("agent", None)
-        actor = Registry.get_agent(
-            agent=actor_fn,
+        actror_conf = config_["actor"]
+        actor = AGENTS.get_from_config(
+            "agent",
```

Comment:
i don't like the idea to duplicate registry name. If we already specified `AGENTS` it's strange to put another key `"agent"`. Also for any other: `CRITERIONS.get_from_config("criterion", ...`, `CALLBACKS.get_from_config("callback", value)`